hasura/graphql-engine
# update console UI components

### Description


- updated RawSQL page to show a larger SQL window
- updated Migrations page, added link to docs
- updated Notifications behavior. 
  - hitting `Details` results is an expanded notification
  - removed expand option to see details in modal

### Affected components 

- Console

### Related Issues




### Solution and Design


Run SQL
![Screenshot from 2019-07-23 19-37-08](https://user-images.githubusercontent.com/153602/61718794-4e722280-ad81-11e9-90f8-ed9ccfe06bab.png)

Migration page
![Screenshot from 2019-07-23 19-38-58](https://user-images.githubusercontent.com/153602/61718939-8d07dd00-ad81-11e9-96ff-65ddec501223.png)

Notification details
![Screenshot from 2019-07-23 19-39-47](https://user-images.githubusercontent.com/153602/61718990-ac066f00-ad81-11e9-8e5e-303aee2b40ef.png)


### Steps to test and verify


- head to RawSQL page
- observe new UI
- hit run without query to cause error
- hit details in error notification
- hit RUN in quick succession to create 2 errors. expand details of any error to see limitation (new details notif will overlap second notification)

### Limitations, known bugs & workarounds

CSS woes
- The expanded notification appears on top of already open notifications
- The message in the expanded notification does not make full use of the space. ie: wraps at the same point as when collapsed
